src/views/PostCreator/components/TopicSelector/TopicSelector.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { AlertInfo } from 'components';
+import { useTopicsQuery } from 'views/PostCreator/PostCreator.hooks';
+
+const TopicSelector = ({ value, handleChange }) => {
+	const { isLoading, isError, error, data, isSuccess } = useTopicsQuery();
+
+	return (
+		<>
+			<Select labelId="topicSelector" id="select" value={value} onChange={handleChange}>
+				<MenuItem value={-1} disabled>
```

Comment:
no i tutaj też, może po prostu puste, ale to w sumie bez jakiejś większej różnicy chyba

Reply:
W sumie tak, dałem te -1 bo bo generalnie value tych topiców jest liczba, ale faktycznie mogłoby być i puste